src/Model/Product/Car.php

```diff
@@ -23,14 +23,15 @@ class Car extends \Pimcore\Model\DataObject\Car
     const OBJECT_TYPE_ACTUAL_CAR = 'actual-car';
     const OBJECT_TYPE_VIRTUAL_CAR = 'virtual-car';
 
-    /**
-     * @return string
-     */
-    public function getOSName(): ?string
+    public function getOSName(): string
```

Comment:
I was thinking of making it consistent with `getName()` but if the usages are not null safe then i would keep it as is..

Reply:
I've found out that i may be wrong in turning it in non-nullable, as in ecommerceframeworkbundle is like this
https://github.com/pimcore/pimcore/blob/b675696e8c3dff29e51d97e67fc79d3003c9ca3a/bundles/EcommerceFrameworkBundle/Model/ProductInterface.php#L34

at least, it should stay nullable even though returning is turned into empty string due the `.` concatenation (unless we add something like
`if ( is_null($this->getManufacturer()) && is_null($this->getName()){ return null; }`  beforehand) but it would be a little more future-proof if `null` need to be used.

wdyt?

Reply:
yes, please make it nullable and just like other cases you can `getOSName` before calling the setters so it should return null.